stac_mjx/io_dict_to_hdf5.py

```diff
@@ -0,0 +1,67 @@
+# copied from https://codereview.stackexchange.com/a/121308 (and slightly modified for updated h5py, Elliott Abe)
```

Comment:
What are you using this file for?


Also, we've been putting this kind of stuff into the `demos` directory, though we will eventually add a directory for utils for stuff like this.

Reply:
This file is how I handle loading and saving h5 files. It is a more flexible way then other approaches I have found since you can easily do nested dictionaries and save them.